src/main/java/com/commercetools/sync/services/impl/BaseService.java

```diff
@@ -59,6 +62,12 @@
 
   private static final int MAXIMUM_ALLOWED_UPDATE_ACTIONS = 500;
   static final String CREATE_FAILED = "Failed to create draft with key: '%s'. Reason: %s";
+  /*
+   * Considering maximum of 256 characters for key and sku fields (key and sku field doesn't have
+   * limit except for ProductType(256)) We chunk them in 60 (keys or sku) we will have around a query
+   * around 15.000 characters. Above this size it could return - Error 414 (Request-URI Too Large)
```

Comment:
Where did you get 414? I also think I saw 413, but I'm not sure. Can you clarify?

Reply:
@butenkor Maybe you know if it was HTTP status 413 or 414? In the docs, I could not find anything.

Reply:
http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers. You can check here for both 413 and 414. We are getting 414, Since it is a get call.

Reply:
I tested it, and it returns 414, so you're right.